### ISSUE TITLE
Footer links for www pages should not end in `/`

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1183]
**TL;DR:** footer links for scratchr2 pages should end in `/`, but footer links for www pages should not.

Pretty much all the footer links for www pages need to be changed (remove the `/` at the end) other than `/developers`. 
`https://day.scratch.mit.edu` does not need to be changed. External links (ScratchEd, ScratchFoundation, etc.) don't need to be changed.

<img width="281" alt="screen shot 2017-02-14 at 5 09 40 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951774/6cc6fbcc-f2d8-11e6-961e-cf75950a84a4.png">
<img width="254" alt="screen shot 2017-02-14 at 5 09 15 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951772/6cc55e3e-f2d8-11e6-8d62-1dcf9e29c919.png">
<img width="292" alt="screen shot 2017-02-14 at 5 09 09 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951776/6cc6f0a0-f2d8-11e6-8aff-7dc7b20fe300.png">
<img width="290" alt="screen shot 2017-02-14 at 5 09 04 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951777/6ccd22f4-f2d8-11e6-9333-24ece900ffae.png">
<img width="249" alt="screen shot 2017-02-14 at 5 08 18 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951775/6cc7514e-f2d8-11e6-8f24-9fa1f123ef79.png">
<img width="322" alt="screen shot 2017-02-14 at 5 08 00 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951773/6cc60ad2-f2d8-11e6-8a77-3a2c373d21a9.png">
<img width="253" alt="screen shot 2017-02-14 at 5 07 47 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951779/6cd1e8f2-f2d8-11e6-8e92-c49bd7092a10.png">
<img width="324" alt="screen shot 2017-02-14 at 5 07 40 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951780/6cd1d24a-f2d8-11e6-9dbd-7a0748705f4a.png">
<img width="285" alt="screen shot 2017-02-14 at 5 07 32 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951778/6cd012ca-f2d8-11e6-97ec-97b71f2fd011.png">
<img width="283" alt="screen shot 2017-02-14 at 5 07 22 pm" src="https://cloud.githubusercontent.com/assets/1320456/22951781/6cd2d80c-f2d8-11e6-9357-7852001fbbfa.png">
